konfera/migrations/0001_initial.py

```diff
@@ -31,14 +31,17 @@ class Migration(migrations.Migration):
             name='Event',
             fields=[
                 ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('date_from', models.DateTimeField(verbose_name='Begining')),
+                ('date_to', models.DateTimeField(verbose_name='End')),
                 ('title', models.CharField(max_length=128, verbose_name='Event name')),
                 ('slug', models.SlugField(help_text='Slug field, relative URL to the event.', verbose_name='Event url')),
                 ('description', models.TextField()),
-                ('date_from', models.DateTimeField(verbose_name='Event begging')),
-                ('date_to', models.DateTimeField(verbose_name='Event end')),
-                ('event_type', models.CharField(choices=[('conference', 'Conference'), ('meetup', 'Meetup')], max_length=20)),
-                ('status', models.CharField(choices=[('draft', 'Draft'), ('published', 'Published'), ('expired', 'Expired')], max_length=20)),
+                ('event_type', models.CharField(choices=[(b'conference', 'Conference'), (b'meetup', 'Meetup')], max_length=20)),
```

Comment:
unfortunately yes, I am using 2 computers and on the other one is python2 default...